exercises/practice/word-count/word_count.go

```diff
@@ -1,7 +1,29 @@
 package wordcount
 
+import (
+	"regexp"
+	"strings"
+)
+
+// Frequency is a map of the frequency of occurrence keyed to the unique word.
 type Frequency map[string]int
 
+// WordCount returns the map of frequency of words based on the input phrase.
 func WordCount(phrase string) Frequency {
-	panic("Please implement the WordCount function")
+	freq := Frequency{}
```

Comment:
This is the stub file, shouldn't contain the solution :)

Reply:
Forgot to add it when testing everything in my local environment